tensorflow_quantum/core/qsim/util.h

```diff
@@ -16,7 +16,10 @@ limitations under the License.
 #ifndef TFQ_CORE_QSIM_UTIL_H_
 #define TFQ_CORE_QSIM_UTIL_H_
 
+#include <bitset>
 #include <cstddef>
+#include <cstdint>
```

Comment:
Why is this needed ?

Reply:
Seems it isn't, at some point I thought it was needed for uint64_t.  Removed.